infrateam_first_responder.md

```diff
@@ -104,11 +111,6 @@ If on a Mac, you will get better results if you stay in the same "space" as the
 
 There are applications that need to be deployed separately (i.e., not using `sdr-deploy`).
 
```

Comment:
I think this sentence should also go??

Reply:
@ndushay I believe this sentence goes with the `Cloud Projects` that now immediately follows it. Should I strike that section and this sentence, given all our DLME and Sinopia work is done by previous steps (merging PRs) or CI? Or should we leave them both?

Reply:
OK, then I guess let's leave them in there for now until we're confident we can stop thinking about them.